Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -13,6 +13,7 @@
 import urllib3
 import io
 import re
+import pickle
```

Comment:
Please don't use pickle. Use json. It is hard to debug this format and also has security concerns.